pages/progress.js

```diff
@@ -15,8 +15,27 @@ function HistoryPage() {
     { orderBy: ['timestamp', 'desc'] }
   );
 
-  return fullyLoaded(user, allUserActivities, completedTasks) ? (
-    <History activities={allUserActivities} completedTasks={completedTasks} />
+  const confidentActivityLogEntries = useUserSubcollection('activityLogEntries', {
+    where: ['activityFeeling', 'array-contains', 'Confident'],
+  });
+
+  const allActivityLogEntries = useUserSubcollection('activityLogEntries', {
```

Comment:
We're already subscribed to `allUserActivities` on line 11 -- can we reuse that?

Reply:
(feel free to add the needed ordering to the existing subcollection subscription)

Reply:
done